server/komodo-rest/src/main/java/org/komodo/rest/relational/RestVdbModelSource.java

```diff
@@ -42,6 +45,23 @@ public RestVdbModelSource() {
         super();
     }
 
+    private KomodoObject findTranslator(UnitOfWork uow, String translatorName, Vdb vdb) throws KException {
+        KomodoObject[] children = vdb.getChildrenOfType(uow, VdbLexicon.Translator.TRANSLATOR);
+        KomodoObject translator = null;
```

Comment:
Why not use `Vdb.getTranslators( uow, translatorName )` here? You won't need to loop and you can return the concrete type of `Translator`.